R/source.R

```diff
@@ -63,12 +65,14 @@ run_script <- function(path = "touchstone/script.R",
 #'
 #' This sets environment variables, R options and library paths to work
 #' interactively on the [touchstone_script].
-#' @param head_branch Git branch to be used as the `GITHUB_HEAD_REF` branch (i.e. the
-#'   branch with new changes) when running benchmarks. Defaults to the current
-#'   branch.
-#' @param base_branch Git branch for the `GITHUB_BASE_REF` (i.e. the branch you want
-#'   to merge your changes into) when running benchmarks.
-#'   Defaults to 'main' if the option `touchstone.default_base_branch`is not set.
+#' @param head_branch Git branch to be used as the `GITHUB_HEAD_REF` branch
+#' (i.e. the branch with new changes) when running benchmarks. Defaults to the
+#' current branch.
+#' @param base_branch Git branch for the `GITHUB_BASE_REF` (i.e. the branch you
+#' want to merge your changes into) when running benchmarks. Defaults to 'main'
+#' if the option `touchstone.default_base_branch`is not set.
+#' @param n Number of times benchmarks should be run for each `branch`. Will
+#' override `n` argument in all interactive calls to [benchmark_run]
```

Comment:
I think tidyverse style compliant {roxygen2} is indented on line break, i.e. 
```r
#' @param x this is param x and the
#'   description does not fit on one line.
NULL
```
Can you adapt it?

Reply:
Also, must end with full stop, and to get code font formatting, we need `[benchmark_run()]`, not `[benchmark_run]` I think.

Reply:
mhm roxygen formatting is always a pain... I wish there was a package that could *style* those comments. :wink: